blockchain/event_filter.go

```diff
@@ -0,0 +1,198 @@
+package blockchain
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/db"
+)
+
+var errChunkSizeReached = errors.New("chunk size reached")
+
+type EventFilter struct {
+	txn       db.Transaction
+	FromBlock uint64
+	ToBlock   uint64
+	From      *felt.Felt
```

Comment:
Personal preference: could we rename `From` to `ContractAddress`? Reads easier in the implementation imho.